src/mobile/package.json

```diff
@@ -43,7 +43,7 @@
     "jsc-android": "241213.x.x",
     "lodash": "^4.17.13",
     "lottie-react-native": "^2.6.1",
-    "nodejs-mobile-react-native": "https://github.com/rajivshah3/nodejs-mobile-react-native#f3e99d92aa6983ca84805839b58cbe2ad2a33a37",
+    "nodejs-mobile-react-native": "0.5.0",
```

Comment:
We cannot use the original version of `nodejs-mobile-react-native` as they don't provide an iOS framework built with bitcode

Reply:
@rajivshah3 Perhaps best to investigate a WebView then.